test/unit/auth/credential.spec.ts

```diff
@@ -395,6 +408,24 @@ describe('Credential', () => {
         privateKey: mockCertificateObject.private_key,
       });
     });
+
+    it('should parse valid token if app def creds point to default refresh token loc', () => {
+      process.env.GOOGLE_APPLICATION_CREDENTIALS = GCLOUD_CREDENTIAL_PATH;
+
+      fsStub = sinon.stub(fs, 'readFileSync').returns(JSON.stringify(MOCK_REFRESH_TOKEN_CONFIG));
+
+      const adc = new ApplicationDefaultCredential();
+      const c = adc.getCredential();
+      expect(c).is.instanceOf(RefreshTokenCredential);
+      expect(c).to.have.property('refreshToken').that.includes({
+        clientId: MOCK_REFRESH_TOKEN_CONFIG.client_id,
+        clientSecret: MOCK_REFRESH_TOKEN_CONFIG.client_secret,
+        refreshToken: MOCK_REFRESH_TOKEN_CONFIG.refresh_token,
+        type: MOCK_REFRESH_TOKEN_CONFIG.type,
+      });
+      // tslint:disable-next-line:no-unused-expression
```

Comment:
Is this required?

Reply:
Odd. My dev environment gives an error on that line if I don't have it, but tslint command line doesn't complain.  Either way I've removed it, updated the test case name as suggested and updated the CHANGELOG